modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp

```diff
@@ -36,7 +36,7 @@ long formInstanceId = ddmFormDisplayContext.getFormInstanceId();
 			</clay:container-fluid>
 		</div>
 	</c:when>
-	<c:when test="<%= ddmFormDisplayContext.isRequireAuthentication() %>">
+	<c:when test="<%= ddmFormDisplayContext.isRequireAuthentication() || (ddmFormDisplayContext.isLimitToOneSubmissionPerUserEnabled() && !ddmFormDisplayContext.isLoggedUser()) %>">
```

Comment:
@marcelabc can you reorder this by putting `isLimitToOneSubmissionPerUserEnabled` and `isLoggedUser` before `isRequireAuthentication`? Thanks!